config.yaml

```diff
@@ -167,6 +167,14 @@ commands:
 
   N98\Magento\Command\Installer\InstallCommand:
     magento-packages:
+      - name: magento-ce-2.1.2
+        package: magento/project-community-edition
+        version: 2.1.2
+        options:
+          prefer-dist: true
+          stability: alpha
+          repository-url: https://repo.magento.com
+    magento-packages:
```

Comment:
Thanks @ktomk
I make the same mistake all the time!
I think is time to change my glasses.
I am gonna ask Santa this Christmas for a new ones